packages/backend/src/open_payments/payment/incoming/service.ts

```diff
@@ -568,3 +579,68 @@ async function addReceivedAmount(
 
   return payment
 }
+
+async function getPage(
```

Comment:
We can have the `getWalletAddressPage` use this function also, just need to add in the wallet address filter in the `options`